lib/core/Cargo.toml

```diff
@@ -14,7 +14,7 @@ frb = ["dep:flutter_rust_bridge"]
 [dependencies]
 anyhow = { workspace = true }
 bip39 = "2.0.0"
-boltz-client = { git = "https://github.com/hydra-yse/boltz-rust", branch = "yse-fee-calculation" }
+boltz-client = { git = "https://github.com/ok300/boltz-rust", branch = "ok300-chain-swap-refund-check-all-utoxs" }
```

Comment:
Do this include @hydra-yse's fee calculation fix?

Reply:
Good point. No, because we started out branches more or less at the same time.

@hydra-yse can your branch be merged upstream? It's a fairly small change and should be easy to test and review.

Reply:
The calculation fix was merged in upstream and in my branch: 87e9327d24e1da02febf3f0e7bba08570111f29e